### DISCUSSION TITLE
Using Vite glob imports in nuxt config

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#29795] [Questions]
I am trying to import some files into my `nuxt.config.ts` file using Vite globs and I keep getting the error:

```
SyntaxError: Cannot use 'import.meta' outside a module
```

`"type": "module"` is in the `package.json` file and I also tried changing the nuxt config extension to `.js` and nothing works.

```js
import path from 'path'
import crypto from 'crypto'

const modules = import.meta.glob('/content/blog**')

export default defineNuxtConfig({ /* ... */ })
```

**Comment by pi0** [comment]
It has to be supported with jiti. Made a tracker: https://github.com/unjs/jiti/issues/339
